### PR TITLE
Handle ConnectionErrors caused by faulty urllib3

### PR DESCRIPTION
**Pull request by tedoc2000**

### CODE REVIEW

**Comment by cory-signalfx** [comment]
Don't forget a version bump and changelog entry!

File: signalfx/ingest.py [hunk 1]
```diff
@@ -267,7 +271,15 @@ def _post(self, data, url, session=None, timeout=None):
             _logger.debug('Compressed payload from %d to %d bytes',
                           uncompressed_bytes, len(data))
 
-        response = session.post(url, data=data, timeout=timeout)
+        try:
+            response = session.post(url, data=data, timeout=timeout)
+        except ConnectionError:
+            if session is self._sesssion:
```
Comment (ricomoss) [inline signalfx/ingest.py:277]: Type:  `_sesssion` -> `_session`

Comment (tedoc2000) [reply signalfx/ingest.py:277]: Thanks! Let me get that fixed..

Comment (tedoc2000) [reply signalfx/ingest.py]: Okay @ricomoss updated.

**Pull request merged**